crates/criticalup-cli/src/commands/install.rs

```diff
@@ -105,6 +109,12 @@ async fn install_product_afresh(
         .await?;
     let verified_release_manifest = release_manifest_from_server.signed.into_verified(&keys)?;
 
+    // Checks for making sure that there is no revoked content in the incoming packages.
+    let revocation_info = &keys
```

Comment:
In the future we'll probably want packages to be a sort of state machine which can't be installed until they are verified, so we don't forget.

Reply:
Added a ticket for this.